src/MiniCover/Reports/Clover/CloverReport.cs

```diff
@@ -0,0 +1,198 @@
+                ));
+        }
+
+        private static IEnumerable<XElement> CreateLinesElement(ICollection<InstrumentedInstruction> instructions, IDictionary<int, int> hits)
+        {
+            return instructions
+                .Select(instruction => new XElement(
+                    XName.Get("line"),
+                    new XAttribute(XName.Get("num"), instruction.StartLine),
+                    new XAttribute(XName.Get("count"), hits.TryGetValue(instruction.Id, out int count) ? count : 0),
+                    new XAttribute(XName.Get("type"), "stmt")
+                ));
+        }
+
+        private static XElement CreateMetricsElement(CloverCounter counter)
+        {
+            var element = new XElement(
+                XName.Get("metrics"),
+                new XAttribute(XName.Get("statements"), counter.Statements),
+                new XAttribute(XName.Get("coveredstatements"), counter.CoveredStatements),
+                new XAttribute(XName.Get("conditionals"), counter.Conditionals),
+                new XAttribute(XName.Get("coveredconditionals"), counter.CoveredConditionals),
+                new XAttribute(XName.Get("methods"), counter.Methods),
+                new XAttribute(XName.Get("coveredmethods"), counter.CoveredMethods),
+                new XAttribute(XName.Get("elements"), counter.Elements),
+                new XAttribute(XName.Get("coveredelements"), counter.CoveredElements)
+            );
+
+            if(counter.Lines > 0)
+            {
+                element.Add(new XAttribute(XName.Get("loc"), counter.Lines));
+                element.Add(new XAttribute(XName.Get("ncloc"), counter.Lines));
+            }
+            if(counter.Classes > 0)
+            {
+                element.Add(new XAttribute(XName.Get("classes"), counter.Classes));
+            }
+            if(counter.Files > 0)
+            {
+                element.Add(new XAttribute(XName.Get("files"), counter.Files));
+            }
+            if(counter.Packages > 0)
+            {
+                element.Add(new XAttribute(XName.Get("packages"), counter.Packages));
+            }
+
+            return element;
+        }
+
+        private static CloverCounter CountProjectMetrics(InstrumentationResult result, IDictionary<int, int> hits)
+        {
+            return result.Assemblies
+                .Select(t => CountPackageMetrics(t, hits))
+                .Aggregate((prev, next) =>
+                {
+                    prev.Add(next);
+                    prev.Packages += 1;
+                    return prev;
+                });;
+        }
+
+        private static CloverCounter CountPackageMetrics(InstrumentedAssembly assembly, IDictionary<int, int> hits)
+        {
+            return assembly.SourceFiles
+                .Select(t => CountFileMetrics(t.Value, hits))
+                .Aggregate((prev, next) =>
+                {
+                    prev.Add(next);
+                    prev.Files += 1;
```

Comment:
++

Reply:
it's a matter of style

Reply:
nope.
increase vs adding 1. 
increase = inc (in IL and in Assembler)
adding = add (registry, value in Assembler)